src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2070,7 +2070,7 @@ internal enum ErrorCode
         ERR_ExpressionTreeContainsUTF8StringLiterals = 9027,
         ERR_MisplacedUnchecked = 9028,
 
-        ERR_FieldKeywordInsideNameOf = 9028,
+        ERR_FieldKeywordInsideNameOf = 9029,
```

Comment:
It would be best to move this well past the current error codes, and put a prototype comment for condensing later. I'd choose something like 9500, otherwise you'll just conflict on the next feature branch merge.